tools/user_migration/gen_bso_users.py

```diff
@@ -90,17 +96,20 @@ def get_users(self, args):
             )
             self.report.fail(uid, "Unexpected error {}".format(ex))
 
-    def run(self):
+    def run(self, bso_num=0):
```

Comment:
nit: I think the bso_num 0 default here is potentially more trouble than it's worth (in case code around it changes later, it's easy to forget about it)

Reply:
Yeah, probably. I prefer that something like that have a default value, but I'm not sure I can strongly defend it here.